tfx/components/infra_validator/model_server_runners/kubernetes_runner_test.py

```diff
@@ -221,6 +226,10 @@ def testBuildPodManifest_InsideKfp_OverrideConfig(self):
                      'my_secret')
     self.assertEqual(container_envs['SECRET'].value_from.secret_key_ref.key,
                      'my_key')
+    container_resources = pod_manifest.spec.containers[0].resources
+    self.assertDictEqual(
+      container_resources.to_dict(), k8s_config_dict.get(
+        "serving_pod_overrides").get("resources"))
```

Comment:
just use `[]` operator as it should present? Also use the single quote string: `k8s_config_dict['serving_pod_overrides']['resources'])`